### PR TITLE
chrony: add multigraph plugin to monitor chrony's delay, freq time and stratum

### PR DESCRIPTION
**Pull request by kimheino**

### CODE REVIEW

File: plugins/chrony/chrony_multigraph [hunk 1]
```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3 -tt
```
Comment (sumpfralle) [inline plugins/chrony/chrony_multigraph:1]: Please use `/usr/bin/env python3` without arguments, if possible.

File: plugins/chrony/chrony_multigraph [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# -*- coding: utf-8 -*-
```
Comment (sumpfralle) [inline plugins/chrony/chrony_multigraph:2]: python2-only, I guess.

File: plugins/chrony/chrony_multigraph [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+"""Munin plugin to monitor chrony NTP daemon status.
```
Comment (sumpfralle) [inline plugins/chrony/chrony_multigraph:4]: Please apply perlpod formatting.

File: plugins/chrony/chrony_multigraph [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    print('delay.label Network path delay')
+
+    print('multigraph chrony_frequency')
+    print('graph_title Chrony clock frequency error')
+    print('graph_vlabel ppm')
+    print('graph_category time')
+    print('graph_args --base 1000')
+    print('frequency.label Local clock frequency error')
+
+    if os.environ.get('MUNIN_CAP_DIRTYCONFIG') == '1':
```
Comment (sumpfralle) [inline plugins/chrony/chrony_multigraph:80]: Maybe move these lines to the main section?

File: plugins/chrony/chrony_multigraph [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        print('multigraph chrony_{}'.format(key))
+        if key == 'stratum' and values[key] == 0:
+            print('{}.value U'.format(key))
+        else:
+            print('{}.value {:.8f}'.format(key, values[key]))
+
+
+if __name__ == '__main__':
+    if len(sys.argv) > 1 and sys.argv[1] == 'autoconf':
+        print('yes' if get_values() else 'no')
```
Comment (sumpfralle) [inline plugins/chrony/chrony_multigraph:97]: It would be nice, if you could distinguish between *something is missing* and *something went wrong*.

**Review by sumpfralle — COMMENTED** [review]
Thank you for this contribution!

Maybe you could think of a less generic name for this plugin (e.g. `chrony_status`)? Or maybe clarify its set of features in the documentation section?

**Comment by kimheino** [comment]
Renamed to chrony_status etc.

**Comment by sumpfralle** [comment]
merged, thanks!

**Pull request merged**